integration-cli/docker_api_network_test.go

```diff
@@ -172,6 +174,53 @@ func (s *DockerSuite) TestAPINetworkConnectDisconnect(c *check.C) {
 	// delete the network
 	deleteNetwork(c, nr.ID, true)
 }
+func delInterface(c *check.C, ifName string) {
```

Comment:
These tests should probably be moved to `integration/` now?

Reply:
We already had this conversation very beginning and I modified testing code so that it can be placed under intergreation-cli .  Pls kindly refer #36054 for old discussion.

Reply:
We do not accept new tests in this package.
Happy to help migrate it over to `integration/network`.

Reply:
I moved test cases to integration/network

Reply:
These are still showing up in integration-cli.